mitodl/mit-open
# Update OpenAPI Generator and Axios

# What are the relevant tickets?
Closes #358 

# Description (What does it do?)
This PR updates OpenAPI Generator to v7.2.0. Previously, the `typescript-axios` generator that we use was tied to axios v0.x; now it uses axios v1, which is good.

# How can this be tested?
There should be no noticeable change in behavior. Check that some API calls are still made correctly:
- front page should load
- as an admin user, you should be able to edit learning paths (click menu in top right corner)
